ethereum/remix-project
# Move scripts to ws-template

This move the scripts to a central place in the code base.

In order to test it:
 -  open FE Hamburger and click on `add Etherscan ...` and `add Deployer ...` this should add the scripts.
 -  open Etherscan plugin, click on `add Helper script`. this should do the same.
